src/urllib3/util/retry.py

```diff
@@ -208,6 +208,7 @@ def __init__(
         allowed_methods: Optional[Collection[str]] = DEFAULT_ALLOWED_METHODS,
         status_forcelist: Optional[Collection[int]] = None,
         backoff_factor: float = 0,
+        backoff_max: float = self.DEFAULT_BACKOFF_MAX,
```

Comment:
Should be `DEFAULT_BACKOFF_MAX` without `self`

Reply:
Fixed!